src/validation/validateLand.ts

```diff
@@ -1,9 +1,16 @@
+import landListe from '../utils/landListe';
+
```

Comment:
Kunne tenke meg å lage koden mer oversiktlig ved å flytte sammen kode som er avhengig av hverandre. Jeg ville ha laget et komponent for bostedsland som rendret Select dropdown og som hadde landlisten inkludert og som exporterte listen.

Reply:
Jeg vil gjerne ha landlisten i en egen fil som nå. Har tenkt til å bruke den andre steder. Enig i resten.

Reply:
Hva med å lage en komponent under /components/bostedsland og legge både komponent og liste der?